CHANGES.md

```diff
@@ -63,6 +63,7 @@
 * We changed the data type for ranges in `JdbcIO.readWithPartitions` from `int` to `long`. This is a relatively minor
     breaking change, which we're implementing to improve the usability of the transform without increasing cruft.
     This transform is relatively new, so we may implement other breaking changes in the future to improve its usability.
+* Support for stopReadTime on KafkaIO SDF (Java).([BEAM-13171](https://issues.apache.org/jira/browse/BEAM-13171)).
```

Comment:
This will likely go under the 2.36 release now since the 2.35 release branch has been cut and is unlikely to take in new features.

Reply:
Copied from the template for the version 2.36.0 and moved the note under it.